documentation/components/tutorial-flow-components-setup.asciidoc

```diff
@@ -139,9 +109,29 @@ When using Maven you can add a the button component into your `pom.xml` as this:
 </dependencies>
 ----
 
-== Maven configuration to exclude webjars
+== Using WebJars
+
+Individual component packages consists of two main sections:
+
+. Java API classes.
+. Web Component files (HTML, JavaScript, CSS ..).
+
+Both sections are provided in the same JAR by default, and both sections are used to display the components correctly in pages.
+
+Web Component files are provided as https://github.com/webjars/webjars/[WebJars] (JAR files that contain Web Component files), the design of which was heavily influenced by the Polymer package manager, https://bower.io/[Bower] (not developed or maintained by Vaadin). WebJars facilitate using Maven, instead of Bower, for the same purpose. 
```

Comment:
Wow, this was rather oddly put (already before the edits). I'll try to rewrite it.

```suggestion
Web Component files are provided as https://github.com/webjars/webjars/[WebJars] (JAR files that contain Web Component files). WebJars contain the resources needed by the client-side and it also contain the dependency information as Maven pom.xml files, so there is no need to use Bower or similar for the client side dependency management. 
```